app/index.html

```diff
@@ -130,18 +130,16 @@
         <div class="iviz-header-left-case">
           <span class="name" style="display: block;">Selected:</span>
           <span class="content">
-            <span id="iviz-header-left-sample-select" @click="openCases('sample')" class="number"
-                  :class="{'iviz-button-hover':highlightAllButtons||highlightCaseButtons}"
-                  role="button" tabindex="0" style="display: block;">{{ selectedSamplesNum }} samples</span>
-            <span>in</span>
-            <span id="iviz-header-left-patient-select" @click="openCases('patient')" class="number"
-                  :class="{'iviz-button-hover':highlightAllButtons||highlightCaseButtons}"
-                  role="button" tabindex="0" style="display: block;">{{ selectedPatientsNum }} patients</span>
+            <span>{{ selectedSamplesNum }} samples / {{ selectedPatientsNum }} patients</span>
           </span>
         </div>
+        <span id="iviz-header-left-patient-select" class="iviz-header-button"
+              @click="openCases('patient')" class="number"
+              :class="{'iviz-button-hover':highlightAllButtons||highlightCaseButtons}"
```

Comment:
We can get rid of highlightAllButtons and highlightCaseButtons if we are not using them.